packages/core/src/components/badge/badge.tsx

```diff
@@ -8,16 +8,24 @@ import clsx from 'clsx';
 })
 export class Badge {
   @Prop()
-  level!: 'primary' | 'success' | 'info' | 'warning' | 'danger';
+  status?: 'primary' | 'success' | 'info' | 'warning' | 'danger';
 
-  @Prop()
-  status?: string;
+  private static statusMap = new Map<string, string>([
+    ['primary', 'Primair'],
+    ['success', 'Gelukt'],
+    ['info', 'Opmerking'],
+    ['warning', 'Waarschuwing'],
+    ['danger', 'Fout']
+  ]);
 
   render() {
     return (
-      <span class={clsx('dso-badge', { [`badge-${this.level}`]: this.level })}>
+      <span class={clsx('dso-badge', { [`badge-${this.status}`]: this.status })}>
         {this.status && (
-          <span class="sr-only">{this.status}: </span>
+          <span class="sr-only">{Badge.statusMap.get(this.status)}: </span>
+        )}
+        {!this.status && (
+          <span class="sr-only">Badge: </span>
```

Comment:
Kopieer voor uw plezier: `<span class="sr-only">{this.status ? Badge.statusMap.get(this.status) : `Badge`}: </span>`

Reply:
Goed gezien!